test/testfiles/t1.qml

```diff
@@ -0,0 +1,121 @@
+import QtQuick 2.6
```

Comment:
This looks like real code. Is it okay to make it open-source? (Or is it already open-source?)

Reply:
I think it's better to be simplified. I'll update it.